lib/matplotlib/contour.py

```diff
@@ -688,6 +689,15 @@ def _find_closest_point_on_leg(p1, p2, p0):
     return d, pc
 
 
+def _is_closed_polygon(X):
+    """
+    Tests whether first and last object in a sequence are the same.  These are
+    presumably coordinates on a polygonal curve, in which case this function
```

Comment:
>>>These are presumably coordinates on a polygonal curve, in which case this function  tests if that curve is closed. 
Sort of odd construction?
"When these are coordinates on a polygonal curve, this function tests if the curve is closed. """